packages/nux/src/components/dot-tip/test/__snapshots__/index.js.snap

```diff
@@ -7,7 +7,7 @@ exports[`DotTip should render correctly 1`] = `
   data-wp-c16t="true"
   data-wp-component="Popover"
   role="dialog"
-  style="position: absolute; top: 0px; left: 0px; opacity: 1; transform: translateX(0px) translateY(0px) translateX(0em) scale(1) translateZ(0); transform-origin: 0% 50% 0;"
+  style="position: absolute; top: 0px; left: 0px; opacity: 1; will-change: auto; transform: translateX(0px) translateY(0px) translateX(0em) scale(1); transform-origin: 0% 50% 0;"
```

Comment:
Are you able to explain why this line needed to change? I guess it's related to the e2e test failures?

Reply:
yes, exactly, seems like the new version of the package is adding this little CSS change so the snapshot tests need to be updated.

Reply:
I think it's pretty benign as changes go. It will change and there's no movement on the Z access. 

I've just merged in trunk to see if that resolves the other E2E failures or futher snapshop updates are needed